js/components/operatorSignIn/list.tsx

```diff
@@ -18,38 +17,35 @@ export const List = ({ line }: { line: HeavyRailLine }): ReactElement => {
   }
 
   return (
-    <div className="m-2">
-      <Link
-        className="inline-block bg-blue text-gray-200 rounded-md p-2 text-sm m-3"
-        to={"/"}
-      >
-        Back
-      </Link>
-      <u>Today&apos;s sign-ins</u>
-      <table>
-        <thead>
-          <tr>
-            <th className="border">Name</th>
-            <th className="border">Badge</th>
-            <th className="border">Time</th>
-            <th className="border">Official</th>
+    <table className="break-words">
+      <colgroup>
+        <col className="w-1/3" />
+        <col className="w-1/5" />
+        <col className="w-1/5" />
+        <col className="w-1/5" />
```

Comment:
These fractions only add to 14/15. I assume flex handles that fine, but I wanna check that this is intentional.

Reply:
Yep 👍 Pedantically this actually isn't a flexbox (unless it inherits, but I don't think it does?), but yes. If anything I wanted to be more lenient than I was being, so I'm going to drop the last 1/5.